### PR TITLE
Updates all exercises to Hapi 9

### LINKED ISSUE #158: update to hapi 9
Probably need to do a bit of a quick run through everything again to get it up to speed for hapi 9.  Now that 9 is out, folks may want to utilize makemehapi to learn how to use it.

### PR DESCRIPTION
**Pull request by fiveisprime**
This updates each of the exercises to use Hapi 9. In most cases it's the
addition of a callback in the #start method, but in others it's the inclusion of
a plugin.

This includes solutions for all French translations as well; however, the French
copy will need some updates/additions.

Adds the following dependencies:

* h2o2: 4.x.x
* inert: 3.x.x
* vision: 2.x.x

Updates the following dependencies:

* hapi: 9.x.x

Closes #158.

### CODE REVIEW

File: exercises/hello_hapi/problem.fr.md [hunk 1]
```diff
@@ -47,6 +47,8 @@ Pour que le serveur commence à écouter sur le port défini au préalable,
 appelez la fonction `start()` :
 
 ```js
-server.start();
+server.start(function () {
+  console.log('Server running at:', server.info.uri);
```
Comment (lloydbenson) [inline exercises/hello_hapi/problem.fr.md]: Might need either fr language here for "Server running at:' or just have it empty.

**Comment by lloydbenson** [comment]
Looks really good to me overall.  Did you get a chance to actually try all these out to validate they were working?

File: exercises/hello_hapi/problem.fr.md (hunk 1 above)
Comment (fiveisprime) [inline exercises/hello_hapi/problem.fr.md]: I went back and forth on that one for a while... I'll just leave it empty.

**Comment by lloydbenson** [comment]
Probably want to make a 3.0.0 milestone and assign it to that.

**Comment by fiveisprime** [comment]
Yes, each one was tested by running through the exercises (both EN and FR). :)

**Comment by lloydbenson** [comment]
Great!  Looks good to me, just make sure its a major version change for obvious reasons.  Thanks for getting that done so quick!

**Comment by lloydbenson** [comment]
Let me know if you want me to merge or if you still have more changes you'd like to do.

**Comment by fiveisprime** [comment]
I'm done with the changes. :)

**Pull request merged**